packages/extension-ui/src/utils/nextDerivationPath.test.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-2020 @polkadot/extension-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AccountJson } from '@polkadot/extension-base/background/types';
+import { nextDerivationPath } from './nextDerivationPath';
+
+const testHierarchy = (accounts: AccountJson[], parentAddress: string, expected: string): void => {
```

Comment:
I believe the tests would be more readable without this method extracted. I suggest inlining it.